lib/money/helpers.rb

```diff
@@ -7,6 +7,7 @@ module Helpers
 
     NUMERIC_REGEX = /\A\s*[\+\-]?\d*(\.\d+)?\s*\z/
     DECIMAL_ZERO = BigDecimal.new(0).freeze
+    MAX_DECIMAL = 20
```

Comment:
we should go for 21 since there was talk of modifying the DB to add an extra decimal (and be `(21,3)`)